modules/backup/variables.tf

```diff
@@ -8,7 +8,7 @@ variable "iam_role_id" {
   type        = string
 }
 
-variable "kms_key_arn" {
+variable "s3_kms_key_arn" {
```

Comment:
Where is this coming from ? I don't see it in main.tf

Reply:
It's fixed, now it will use calculated value in the main.tf